rosbag2_py/CMakeLists.txt

```diff
@@ -40,6 +40,30 @@ if(WIN32 AND CMAKE_BUILD_TYPE STREQUAL "Debug")
   set(PYTHON_LIBRARIES "${PYTHON_DEBUG_LIBRARIES}")
 endif()
 
+function(clean_windows_flags target)
+  # More hacks to avoid pybind11 issues, similar to (but not the same)
```

Comment:
It'd be nice to have a TODO here to remove after a fix upstream.

Reply:
Done https://github.com/ros2/rosbag2/pull/538/commits/354c3abc8b35ef23286e565fc0c9ce77d3d7d1d7.